runtime/core/src/test/java/io/atlasmap/converters/BooleanConverterTest.java

```diff
@@ -58,9 +57,12 @@ public void convertToBooleanNull() throws Exception {
         assertNull(t2);
     }
 
-    @Test(expected = AtlasConversionException.class)
+    @Test
     public void convertToByte() throws Exception {
-        converter.convertToByte(Boolean.TRUE);
+        Byte trueValue = (byte) 1;
+        assertEquals(trueValue, converter.convertToByte(Boolean.TRUE));
+        Byte falseValue = (byte) 0;
```

Comment:
maybe worth adding more complex byte such as (byte) 0.1, Given the current code it should return True (and seems to be the expected result from classical Java implementations) What do you think?